### DISCUSSION TITLE
Changing Grafana Default Port

### DISCUSSION THREAD

**Discussion by breezin101** [teslamate-org/teslamate#4149] [General]
Changed the port to 5000 in the config:

![image](https://github.com/user-attachments/assets/c3e179d8-d7ed-4226-8978-76d64713c003)

Followed the steps to upgrade. But I can't seem access grafana on port 5000:

![image](https://github.com/user-attachments/assets/d3aa6850-2797-4cba-8500-b5f3fe966fea)

I still see 3000/tcp in there... did I miss something?

Thanks!

**Comment by cwanja** [comment]
I think it has to be `5000:3000` in the config. Believe it goes `host:container`

**Reply by breezin101** [reply]
Thanks that did it! Was driving me nuts.
